mmhuman3d/utils/transforms.py

```diff
@@ -546,3 +546,146 @@ def _sja_to_aa(sja, R_t, R_t_inv):
         raise ValueError(f'Invalid input R_t_inv shape f{R_t.shape}.')
     t = Compose([_sja_to_aa])
     return t(sja, R_t=R_t, R_t_inv=R_t_inv)
+
+
+def transform_to_camera_frame(global_orient, transl, pelvis, extrinsic):
```

Comment:
We may consider putting `transform_to_camera_frame` and `batch_transform_to_camera_frame` back to `mmhuman3d/models/body_models/utils.py` for two reasons: 

1. the `transforms.py` contains very general rotation convertions (e.g. axis-angle to rotmat), but  `transform_to_camera_frame` and `batch_transform_to_camera_frame` are very specific to body models.
2. some of our downstream projects are using `transform_to_camera_frame` and `batch_transform_to_camera_frame`, changing their locations may cause incompatibility

What do you think?

Reply:
> the transforms.py contains very general rotation convertions (e.g. axis-angle to rotmat), but transform_to_camera_frame and batch_transform_to_camera_frame are very specific to body models.

I was considering to move it back, but yes, these transforms are very specific to body models.

> some of our downstream projects are using transform_to_camera_frame and batch_transform_to_camera_frame, changing their locations may cause incompatibility

This is even stronger reason. ✅